04-store-api/starter/routes/products.js

```diff
@@ -0,0 +1,10 @@
+const express = require('express');
+const { join } = require('lodash');
```

Comment:
Requires change:
Dead import that is not being used. This import can be removed.